pkg/clients/cluster/cluster_test.go

```diff
@@ -1441,48 +1439,6 @@ func TestIsUpToDate(t *testing.T) {
 				isErr:    false,
 			},
 		},
-		"UpToDateIgnoreRefs": {
```

Comment:
I don't think these were actually doing anything anymore, as IsUpToDate doesn't seem to treat references specially.